sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/DefaultTaskSpec.java

```diff
@@ -44,6 +51,14 @@
     @Valid
     private final Collection<ConfigFileSpec> configFiles;
 
+    @DecimalMin("0")
+    @DecimalMax("1209600") //<< two weeks
```

Comment:
Why is this the max?  Are we avoiding overflow since it's unit is nanoseconds?  It's fine if it's just a reasonable max as well.